doc/Core.xml

```diff
@@ -5369,6 +5394,384 @@ onvif://www.onvif.org/name/ARV-453
+            </listitem>
+            <listitem>
+              <para role="param">LockPasswordPolicy - optional [xs:boolean]</para>
+              <para role="text">Prevent future update of the policy until the next factory default</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>faults</term>
+            <listitem>
+              <para role="text">No command specific faults defined</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>access class</term>
+            <listitem>
+              <para role="access">READ_SYSTEM</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+      </section>
+      
+      <section>
+        <title>SetPasswordComplexityConfiguration</title>
+        <para>A device signaling support for modifying the password complexity policy via the PasswordComplexitySupported capability shall support this operation to set the password policy configuration. If LockPasswordPolicy is supported and is set to true, the current configuration will be locked and will be unlocked only via factory default reset.</para>
+        <variablelist role="op">
+          <varlistentry>
+            <term>request</term>
+            <listitem>
+              <para role="param">MinLen - optional [xs:integer]</para>
+              <para role="text">Minimum password length</para>
+            </listitem>
+            <listitem>
+              <para role="param">Uppercase - optional [xs:integer]</para>
+              <para role="text">Minimum number of uppercase characters</para>
+            </listitem>
+            <listitem>
+              <para role="param">Number - optional [xs:integer]</para>
+              <para role="text">Minimum number of numeric characters</para>
+            </listitem>
+            <listitem>
+              <para role="param">SpecialChars - optional [xs:integer]</para>
+              <para role="text">Minimum number of non-alphanumeric characters</para>
+            </listitem>
+            <listitem>
+              <para role="param">BlockUsernameOccurrence - optional [xs:boolean]</para>
+              <para role="text">Blocks the use of the username in the password</para>
+            </listitem>
+            <listitem>
+              <para role="param">LockPasswordPolicy - optional [xs:boolean]</para>
+              <para role="text">Prevent future update of the policy until the next factory default</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>response</term>
+            <listitem>
+              <para role="text">This is an empty message.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>faults</term>
+            <listitem>
+              <para role="param">env:Sender - ter:InvalidArgVal - ter:ServiceNotSupported</para>
```

Comment:
This error code is used for "The supplied network service is not supported." which doesn't seem right here. Probably you can just remove that part.

Reply:
We could use the generic "InvalidArgVal, Argument Value Invalid" to signal that the configuration settings are not correct

Reply:
Yes I think that would be fine